cmd/envtool/tests_test.go

```diff
@@ -68,11 +68,12 @@ func toLogLines(buf *bytes.Buffer) []string {
 func bufLogger() (*slog.Logger, *bytes.Buffer) {
 	var buf bytes.Buffer
 	h := logging.NewHandler(&buf, &logging.NewHandlerOpts{
-		Base:          "console",
-		Level:         slog.LevelInfo,
-		RemoveTime:    true,
-		RemoveSource:  true,
-		CheckMessages: false, // TODO https://github.com/FerretDB/FerretDB/issues/4511
+		Base:         "console",
+		Level:        slog.LevelInfo,
+		RemoveTime:   true,
+		RemoveSource: true,
+		// the logger handles messages from the `go test` command output
```

Comment:
But those are tests we write correct?

Reply:
`go test` output is not only the logs that we produce. It's everything from `=== RUN`/`--- PASS:` prefix to informations about code coverage etc. We don't produce this output.

Reply:
ok!